src/SimpleCalculator.Web/Services/CalculationService.cs

```diff
@@ -0,0 +1,63 @@
+using Calculations;
+using AutoMapper;
+using SimpleCalculator.Web.Services.Interfaces;
+using SimpleCalculator.DataAccess.Data;
+using SimpleCalculator.DataAccess.Model;
+using SimpleCalculator.Web.Models;
+namespace SimpleCalculator.Web.Services;
+
+public class CalculationService : ICalculationService<CalculationInputModel>
+{
+    private readonly IAddition _addition;
+    private readonly ISubtraction _subtraction;
+    private readonly IDivision _division;
+    private readonly IMultiplication _multiplication;
+    private readonly IMapper _mapper;
+    private readonly CalculatorDbContext _context;
+
+    public CalculationService(IMultiplication multiplication, IDivision division, ISubtraction subtraction,
+        IAddition addition, CalculatorDbContext context, IMapper mapper)
+    {
+        _addition = addition;
+        _context = context;
+        _division = division;
+        _mapper = mapper;
+        _multiplication = multiplication;
+        _subtraction = subtraction;
+    }
+    public double AddService(CalculationInputModel model)
```

Comment:
I think it doesn't make sense to add `Service` part to method's names. Service is structural entity, methods do business logic and should be named by exact actions